### ISSUE TITLE
k3s version - errors on deploying rbac.yaml

### ISSUE THREAD

**Issue by lachnerd** [sleighzy/k3s-traefik-v2-kubernetes-crd#20]
Hi,

im getting follwing errors on applying 001-rbac.yaml to my k3s cluster version v1.19.5+k3s2.

```
---
Error from server (Invalid): error when creating "https://raw.githubusercontent.com/sleighzy/k3s-traefik-v2-kubernetes-crd/master/001-rbac.yaml": CustomResourceDefinition.apiextensions.k8s.io "ingressroutes.traefik.containo.us" is invalid: [spec.versions[0].schema.openAPIV3Schema: Required value: schemas are required, spec.versions[1].schema.openAPIV3Schema: Required value: schemas are required]
...
```

Do you have any clue whats the reason for this ?

**Comment by sleighzy** [comment]
Hi @lachnerd   

In https://github.com/sleighzy/k3s-traefik-v2-kubernetes-crd/commit/2369d11425400d925e078104b99d54d4a00e1567 that file was updated to remove deprecation warnings about the Kubernetes API version. See https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/ for more information on the versioning. 

You can try applying the previous version of that 001-rbac.yaml file before I introduced the change and that error shouldn’t occur. 

I’ll look into what is required for resolving the error as soon as possible.

**Comment by sleighzy** [comment]
The Traefik feature request to update the CRDs was raised in https://github.com/traefik/traefik/issues/5473 and merged in https://github.com/traefik/traefik/pull/7815. This looks to be included in the as yet unreleased Traefik 2.5 version. I’ll update my 001-rbac.yaml to match this in the next few days.
